src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/dialog/clone/CloneRadDialog.java

```diff
@@ -120,11 +121,17 @@ private void setActiveProfile() {
             return;
         }
         ApplicationManager.getApplication().executeOnPooledThread(() -> {
-            var radSelf = new RadSelf(RadSelf.RadSelfAction.ACTIVE_PROFILE);
+            var radSelf = new RadSelf("", "");
```

Comment:
This is confusing.
You're passing empty args for home and path, which are only used by rad service as default values if there aren't any in settings.
I think that these default values should be optional (use an empty constructor, instead of passing invalid values) and handle errors accordingly/meaningfully